en/docs/learn/configuring-data-sharing-for-secondary-users-configuration.md

```diff
@@ -0,0 +1,14 @@
+# Configuration
+
+## Enabling Secondary User Accounts
+
+Follow the below instructions to enable the secondary user account feature:		
+
+1. Open the `<IS_HOME>/repository/conf/deployment.toml` file.
+
+2. Add the following configurations:
+
+    ```toml
```

Comment:
remove "toml"

Reply:
This is required for the docs formatting.